static/js/app.js

```diff
@@ -276,10 +276,54 @@ $(function() {
             return params;
         },
         setMultipleMarkersIcon: function() {
+            var markerGroups = [];
+            var markerObj;
+            var groupID = 1;
+            var groupsColours = [];
             _.each(this.oms.markersNearAnyOtherMarker(), function(marker) {
                 marker.icon = getIcon("multiple", SEVERITY_VARIOUS);
```

Comment:
Are you referring to:
`marker.icon = getIcon("multiple", SEVERITY_VARIOUS);`

If so, than I thought it should be there as a fallback(because that was the base).
You think it's redundant?